### DISCUSSION TITLE
Update Angular 16

### DISCUSSION THREAD

**Discussion by lrm6666** [ShankyTiwari/ng-material-multilevel-menu#230] [General]
Hi,
Are you planning to release a version compatible with angular 17?
Thank you in advance

**Comment by packe100** [comment]
Hi, it seems that the project maintenance is quite unavailable.

Because I didn't receive any answer too, I decided to build my own fork to make this useful library compatible wih Angular18.
The repo is here: https://github.com/packe100/ng-material-multilevel-menu
I'm going to migrate for Angular 20 too.

@ShankyTiwari I don't know how it works, thisi is my first fork. Is it possible to merge to the main project?
